src/ILVerify/src/VerificationResult.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace ILVerify
+{
+    public class VerificationResult
+    {
+        public string ModuleName { get; internal set; }
```

Comment:
Do we need ModuleName here? The verification granularity is on module, so it will be always same for the given invocation.

If we need it here, it should be `PEReader` that the caller gets the ModuleName from if it needs it.